### ISSUE TITLE
can not start telegram cli-help please

### ISSUE THREAD

**Issue by smn200844** [vysheng/tg#730]
Hi
when I enter this command (telegram -k /etc/telegram/tg.pub) to run telegram cli, this message will be appear 
 *** Connect with 173.240.5.1:443 timeout
 *** Can not create connection to DC. Is network down?
----------
what should I do?
the result ping is "100% lost" , even in ping.eu. and all of my VPS's , should I use  173.240.5.1 certainly? it is not possible to change it?
Please help me , Thanks

**Comment by LukeLR** [comment]
If you have a 100% ping loss, this clearly looks like your internet connection. Are you sure you are online?

**Comment by smn200844** [comment]
I am online that I can send this message.you can try it your self,
ping it on www.ping.eu
you will see 100% loss. and even if you check 443 port , it is closed
on this server,you really have access to this server? where are you
from

**Comment by LukeLR** [comment]
Yes, I also cannot reach this ip, but AFAIK it does not even below to telegram's current data centers. Are you sure you are on the latest version of TG-cli?
